modules/pages.py

```diff
@@ -103,9 +105,8 @@ def predicate(x, key):
 
 
 def make_get_blog_entries_by_bird(builder):
```

Comment:
Now that we're not doing anything in the parent `make_*` method, is there a simpler way to do this in Python? In JS, I would use `bind` to create a partial. Oh hey, I found it; pushing... :)